tests/test_cameraMapper.py

```diff
@@ -218,6 +218,12 @@ def testNames(self):
 class Mapper2TestCase(unittest.TestCase):
     """A test case for the mapper used by the data butler."""
 
+    def setUp(self):
+        super().setUp()
```

Comment:
Is this ever necessary for a direct inheritor from `unittest.TestCase`?

Reply:
Maybe not, but it's good practice in general, and does no harm here.